sys/arm64/arm64/pmap.c

```diff
@@ -4356,6 +4375,17 @@ pmap_copy_page(vm_page_t msrc, vm_page_t mdst)
 	vm_offset_t src = PHYS_TO_DMAP(VM_PAGE_TO_PHYS(msrc));
 	vm_offset_t dst = PHYS_TO_DMAP(VM_PAGE_TO_PHYS(mdst));
 
+#if __has_feature(capabilities)
+	pagecopy_cleartags((void *)src, (void *)dst);
```

Comment:
Has the model been updated for tag-stripping on missing PERM_LOAD_CAP? If so we could change this to be a capability-based pagecopy without LOAD_CAP?

Reply:
It's probably not worth stressing about this.  I'm about to kill page_copy_tags() entirely over in the `VM_PROT_CAP` branch and have `pmap_copy_page` always preserve tags.